internal/log/settings.go

```diff
@@ -0,0 +1,124 @@
+package log
+
+import (
+	"strings"
+)
+
+const (
+	meta = 1 << iota
+	success
+	failure
+	summary
+	skips
+	execution
+	executionOutput
+	executionInfo
+	emptySummary
+	enableAll = ^0 // Set all bits as 1
+)
+
+type Settings int16
```

Comment:
We can name it as `OutputSettings` if we rename the `SettingsInterface` to just `Settings`

Reply:
done